packages/arb-token-bridge-ui/tests/e2e/specs/depositETH.cy.ts

```diff
@@ -1,15 +1,19 @@
 /**
- * When user wants to bridge ETH from L1 to L2
+ * When user wants to bridge native token from L1 to L2
```

Comment:
Should we rename this file to `depositNativeToken.cy.ts` ?

Reply:
Updated